### ISSUE TITLE
Blocked on login screen (running locally)

### ISSUE THREAD

**Issue by Zulko** [LeafLIMS/LeafLIMS#6]
Not sure if missing something obvious, but the "admin"/password doesnt work in my locally deployed LeafLIMS. Here is what I have in my ``.env`` (produced by installer.py):

```
SETUP_ADMIN_PASSWORD=pw
```
But I can't get passed the login screen with admin/pw:

![image](https://user-images.githubusercontent.com/1646122/41352482-7d88146c-6f11-11e8-9162-fcc94934fe53.png)

Here is what the docker console says when I log in:

![image](https://user-images.githubusercontent.com/1646122/41352567-ba6378ea-6f11-11e8-97aa-0e75380d87ba.png)

**Comment by tjomasc** [comment]
I've tested it locally using 3.3 and I can't seem to replicate the issue. Have you set up a version of Leaf LIMS before? If so the old volume that contains previous setup data may still be present and you'll have to delete it (See here for more information: https://gist.github.com/bastman/5b57ddb3c11942094f8d0a97d461b430). If you could use the developer tools to provide me with the response from the server that would be helpful as well.

A work around is to try starting manually for now with a different project and checking if that works: `docker-compose -p leaflims2 up --build`.

**Comment by Zulko** [comment]
The command ``docker-compose -p leaflims2 up --build`` did solve the issue. Thanks. It would be nice to have some quick "how to" recipes in the docs to e.g. change an admin password etc.
